odl/solvers/util/callback.py

```diff
@@ -482,8 +489,20 @@ def __init__(self, *args, **kwargs):
         self.kwargs = kwargs
         self.fig = kwargs.pop('fig', None)
         self.step = kwargs.pop('step', 1)
-        self.saveto = kwargs.pop('saveto', None)
-        self.title = kwargs.pop('title', 'Iterate {}')
+        saveto = kwargs.pop('saveto', None)
+        try:
+            saveto = str(saveto + '')
+            self.saveto = saveto.format
+        except TypeError:
+            self.saveto = saveto
+
+        title = kwargs.pop('title', 'Iterate {}')
+        try:
+            title = str(title + '')
+            self.title = title.format
+        except TypeError:
+            self.title = title
```

Comment:
Added an extra `title_formatter` attribute that solves this and the above comment.